XenModel/XenAPI/VM.cs

```diff
@@ -3798,6 +3798,20 @@ public virtual vm_power_state power_state
         private vm_power_state _power_state;
 
         /// <summary>
+        /// Call a XenAPI plugin on this vm

```

Comment:
Here and in Proxy.cs, please use the SDK that has been built by the build system. That will ensure that the calls don't move later.